src/js/components/common/enterpriseNotification.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { InfoOutlined as InfoOutlinedIcon } from '@material-ui/icons';
+
+const EnterpriseNotification = props => {
+  return (
+    !props.isEnterprise && (
+      <p className="info icon">
+        <InfoOutlinedIcon fontSize="small" style={{ verticalAlign: 'middle', margin: '0 6px 4px 0' }} />
+        {`Hosted Mender & Mender Enterprise users can ${props.benefit}. ${<a href="https://mender.io">Learn more</a>}`}
```

Comment:
There is an [object Object] at the end of each message, which I guess is caused by this link

Reply:
should be fixed now 👌